.github/workflows/ci-pull.yml

```diff
@@ -18,10 +18,36 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    #services: TODO add couchdb container for testing
+    services: 
+      cloudant:
+        #Update Readme to match
+        image: couchdb:latest
```

Comment:
This should use a specific couchdb version, not latest, otherwise the build can break unexpectedly. Dependabot will notify of any updates